reports/proposal.md

```diff
@@ -1,8 +1,55 @@
+# STAA 556: Visual Project Proposal
+
+#### Team Members
+- [Rodney Murray](mailto:Rodney.Murray@colostate.edu)
+- [Brendon Stanley](mailto:Brendon.Stanley@rams.colostate.edu)
+
 ## Introduction
-* Context and background information
-* Study objectives
-* Experiment design
-* Other details
+
+<p style="text-indent: 40px;">
+    Hurricane Ian swept through the Hurricane Belt in 2022 leaving behind 149 deaths in Florida
+and an estimated $113 Billion USD in damages (Florida Department of Law Enforcement; National Centers for Environmental Information).
+Concerns were raised regarding the readability of the hurricane forecast maps and the "cone" used to predict the path of the hurricane (Dance and Ajasa).
```

Comment:
Maybe reword to soemthing like:

the "cone" is used to show the predicted path...